bot.ts

```diff
@@ -36,90 +57,158 @@ const commands: SlashCommandPartial[] = [
 		options: [
 			{
 				name: "game",
-				description: "A game's abbreviation",
+				description: "A game's URL",
```

Comment:
The settings page does call it that. And I'd prefer URL, since abbreviation is what API v1 calls it, and unless someone has worked with that, then they could be confused. And that I've seen someone be confused by calling it an abbreviation before.